Source/Editor/GUI/Dialogs/ColorPickerDialog.cs

```diff
@@ -319,7 +320,7 @@ public override void Draw()
         protected override void OnShow()
         {
             // Auto cancel on lost focus
-            ((WindowRootControl)Root).Window.LostFocus += OnCancel;
+            //((WindowRootControl)Root).Window.LostFocus += OnCancel;
```

Comment:
Wrap it around `#ifdef !PLATFORM_LINUX`/`#endif` instead.

Reply:
Done.